### PR TITLE
carina api tests were created

### PR DESCRIPTION
**Pull request by olpapina**

### CODE REVIEW

File: test/getrequest.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+describe('GET method tests', () => {
+        let rawdata = fs.readFileSync('./test/resources/get_rs.json');
+        let expectedData = JSON.parse(rawdata);
+        let expectedSchema = JSON.parse(fs.readFileSync('./test/resources/get_rs.schema'));
+        it('Return 10 users from https://jsonplaceholder.typicode.com/', async function () {
+                await axios({
+                        method: 'GET',
+                        url: 'https://jsonplaceholder.typicode.com/users',
+                        params: {
+                                _limit: 10
```
Comment (twikssi) [inline test/getrequest.test.js]: better to put value 10 to file or declare it with "let" for avoiding hardcoding

File: test/postrequest.test.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            method: 'POST',
+            url: 'https://jsonplaceholder.typicode.com/users',
+            data: postRequest
+        })
+            .then(function (response) {
+                let expectedDataString = JSON.stringify(expectedResponse);
+                let actualDataString = JSON.stringify(response.data);
+                assert.equal(response.status, 201);
+                assert.equal(response.statusText, 'Created');
+                assert.deepEqual(response.data.id, 11);
```
Comment (twikssi) [inline test/postrequest.test.js]: better to put value 11 to file or declare it with "let" for avoiding hardcoding

File: test/postrequest.test.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        delete postRequest.username;
+        await axios({
+            method: 'POST',
+            url: 'https://jsonplaceholder.typicode.com/users',
+            data: postRequest
+        })
+            .then(function (response) {
+                assert.equal(response.status, 201);
+                assert.equal(response.statusText, 'Created');
+                assert.deepEqual(response.data.id, 11);
```
Comment (twikssi) [inline test/postrequest.test.js]: better to put value 11 to file or declare it with "let" for avoiding hardcoding

File: test/getrequest.test.js (hunk 1 above)
Comment (olpapina) [reply test/getrequest.test.js]: fixed for all comments in bc0285d

**Pull request merged**